deploy-gae-build-task/run.ts

```diff
@@ -58,13 +61,13 @@ async function run(): Promise<void> {
   const endpoint = new Endpoint(endpointId);
   await deployGae({
```

Comment:
Nit, since the call to `deployGae` is growing so much, would it be possible at some point to use the named fields instead? To make sure that we're setting the right values? Positional parameters only scale so much.

Reply:
These are in fact named fields, not positional parameters.